src/main/kotlin/gropius/model/issue/Issue.kt

```diff
@@ -0,0 +1,169 @@
+package gropius.model.issue
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import gropius.model.architecture.AffectedByIssue
+import gropius.model.architecture.IMSProject
+import gropius.model.architecture.Trackable
+import gropius.model.common.SyncNode
+import gropius.model.issue.timeline.*
+import gropius.model.template.IssuePriority
+import gropius.model.template.IssueType
+import gropius.model.user.User
+import io.github.graphglue.model.*
+import org.springframework.data.annotation.Transient
+import java.time.Duration
+import java.time.OffsetDateTime
+
+// TODO keep isDuplicate?
+// TODO add (Un)MarkedAsDuplicateEvents?
+// TODO reintroduce Referenced Events?
+@DomainNode
+@GraphQLDescription(
+    """An Issue in the Gropius system.
+    Issues can be used to report bugs, request features, ask questions, ...
+    Issues are synced to all IMSProjects of Trackables they are part of.
+    All changes to the Issue are reflected by the timeline.
+    """
+)
+class Issue(
+    createdAt: OffsetDateTime,
+    lastModifiedAt: OffsetDateTime,
+    @GraphQLDescription("Title of the Issue, usually a short description of the Issue.")
+    @FilterProperty
+    @OrderProperty
+    var title: String,
+    @GraphQLDescription("The DateTime when the Issue was last updated, this includes a changed timeline.")
+    @FilterProperty
+    @OrderProperty
+    var lastUpdatedAt: OffsetDateTime,
+    @GraphQLDescription("If true, this Issue is currently open, otherwise it is closed")
+    @FilterProperty
+    var isOpen: Boolean,
+    @GraphQLDescription("DateTime when working on this Issue started / will start.")
+    @FilterProperty
+    @OrderProperty
+    var startDate: OffsetDateTime?,
+    @GraphQLDescription("DateTime when working on this Issue should be finished.")
+    @FilterProperty
+    @OrderProperty
+    var dueDate: OffsetDateTime?,
+    @GraphQLDescription("Estimated amount of time necessary for this Issue.")
+    @FilterProperty
+    @OrderProperty
+    var estimatedTime: Duration?,
+    @GraphQLDescription("Time spent working on this Issue.")
+    @FilterProperty
+    @OrderProperty
+    var spentTime: Duration?
```

Comment:
By whom? This looks overly overspecified for features we do not really support. Maybe change this into a function which computes it from timeline items? Don't want to sync a number

Reply:
If we change this to a function, we would always have to load those TimelineItems which is 
- rather costly
- destroys our one-db-query principle
Therefore I would not recommend this approach for the property in our db.
However, for the sync, you can compute this via a function if you want to (or do not sync it at all).